src/stan/agrad/rev/matrix/value_of.hpp

```diff
@@ -23,12 +23,11 @@ namespace stan {
      *
      * This is used as a convenience function for implementing varis of matrix operations.
      **/
-    template<int R,int C>
-    inline Eigen::Matrix<double,R,C> value_of(const Eigen::Matrix<var,R,C> &M) {
-      int i,j;
+    template<int R, int C>
+    inline Eigen::Matrix<double,R,C> value_of(const Eigen::Matrix<var,R,C>& M) {
```

Comment:
This whole definition should go away  It's redundant given the template function in

`src/stan/math/matrix/value_of_rec.hpp`